test/crypto.js

```diff
@@ -0,0 +1,135 @@
+/* eslint-env mocha */
+/* global MasqCommon */
+/* global chai */
+
+describe('MasqCommon utils', function () {
+  context('Generating a random buffer (e.g. iv)', () => {
+    it('Should generate a random buffer without length parameter', () => {
+      const iv1 = MasqCommon.crypto.genRandomBuffer()
+      chai.assert.lengthOf(iv1, 16, 'Default length is 16')
+    })
+    it('Should generate a random buffer with a specific length parameter', () => {
+      const iv2 = MasqCommon.crypto.genRandomBuffer(8)
+      chai.assert.lengthOf(iv2, 8, 'Array length is not 8')
+    })
+  })
+  context('Should derive and hash a passphrase ', () => {
+    let passphrase = 'mySecretPass'
+    it('Should derive a passphrase [string] with default settings', async () => {
+      const hashedPassphrase = await MasqCommon.crypto.derivePassphrase(passphrase)
+      chai.assert.equal(hashedPassphrase.hashAlgo, 'SHA-256', 'Default hash algo is SHA-256')
+      chai.assert.equal(hashedPassphrase.iterations, 100000, 'Default iteration is 100000')
+      chai.assert.lengthOf(hashedPassphrase.storedHash, 64, 'SHA-256 returns a 256 bits array, 64 bytes as hex string')
+      chai.assert.lengthOf(hashedPassphrase.salt, 32, 'Default salt is 128 bits array, 32 bytes as hex string')
+    })
+    it('Should reject if passphrase is not a string or is empty', async () => {
```

Comment:
can you add a newline before each context, describe, it please ?

Reply:
done.